headers/functions/overlay29.h

```diff
@@ -392,12 +412,24 @@ bool IsInSpawnList(undefined* spawn_list, enum monster_id monster_id);
 int ChangeShayminForme(struct entity* entity, int forme);
 void ApplyItemEffect(undefined4 param_1, undefined4 param_2, undefined4 param_3,
                      struct entity* attacker, struct entity* defender, struct item* thrown_item);
+void ApplyCheriBerryEffect(struct entity* attacker, struct entity* defender);
+void ApplyPechaBerryEffect(struct entity* attacker, struct entity* defender);
+void ApplyRawstBerryEffect(struct entity* attacker, struct entity* defender);
+void ApplyHungerSeedEffect(struct entity* attacker, struct entity* defender);
+void ApplyVileSeedEffect(struct entity* attacker, struct entity* defender);
 void ViolentSeedBoost(struct entity* attacker, struct entity* defender);
+void ApplyGinsengEffect(struct entity* attacker, struct entity* defender);
+void ApplyBlastSeedEffect(struct entity* attacker, struct entity* defender, bool thrown);
 void ApplyGummiBoostsDungeonMode(struct entity* user, struct entity* target,
                                  enum type_id gummi_type, int random_stat_boost);
 bool CanMonsterUseItem(struct entity* entity, struct item* item);
-void ApplyGrimyFoodEffect(struct entity* user, struct entity* target);
-void ApplyMixElixirEffect(struct entity* user, struct entity* target);
+void ApplyGrimyFoodEffect(struct entity* attacker, struct entity* defender);
```

Comment:
All the functions in this block should be `user` and `target`, not `attacker` and `defender` to match what the symbol descriptions say.